plugins/inputs/webhooks/github/github_webhooks.go

```diff
@@ -29,6 +33,13 @@ func (gh *GithubWebhook) eventHandler(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusBadRequest)
 		return
 	}
+
+	if gh.Secret != "" && !checkSignature(gh.Secret, data, r.Header["X-Hub-Signature"][0]) {
```

Comment:
This will panic if the header `X-Hub-Signature` doesn't exist. It's recommended to use `r.Header.Get("X-Hub-Signature")` instead.

Same thing with the `X-Github-Event` up above. I know it's pre-existing code, but would probably be good time to fix it.